doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -1134,8 +1126,13 @@ private function is_valid_token($step)
     }
 
     private function save_api_settings($api_settings)
-    {
-        extract($api_settings);
```

Comment:
This is the one of the best changes, since this function should be avoided :clap: